electron/src/main.ts

```diff
@@ -223,6 +223,7 @@ const showMainWindow = (mainWindowState: WindowStateKeeper.State) => {
         main.hide();
       }
     }
+    systemMenu.unregisterShortcuts();
```

Comment:
Will it trigger a `focus` when the app is opened again? Else, it seems the shortcuts will never be registered again.

Reply:
Yes, the `focus` event is triggered when clicking on the taskbar icon.